src/diameter_handlers.cpp

```diff
@@ -483,11 +494,16 @@ void PushProfileTask::on_get_ims_sub_failure(Store::Status rc)
   delete this;
 }
 
-void PushProfileTask::on_save_ims_sub_success()
+void PushProfileTask::on_save_ims_sub_progress()
 {
   SAS::Event event(this->trail(), SASEvent::CACHE_PUT_REG_DATA_SUCCESS, 0);
   SAS::report_event(event);
   send_ppa(DIAMETER_REQ_SUCCESS);
+}
+
+void PushProfileTask::on_save_ims_sub_success()
+{
+  // Just tidy up
```

Comment:
Should we be sending a SAS log when the handler finishes an action?

Reply:
Hmm, possibly, but the memcachedcache will be logging to SAS when it's doing stuff so I'm inclined to leave it as-is.